tests/pytest_framework/src/syslog_ng_config/statements/filters/filter.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2015-2018 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+from src.common.random_id import RandomId
+
+
+class Filter:
+    def __init__(self, logger_factory, instance_paths, **kwargs):
+        self.options = kwargs
+
+        self.driver_id = "driverid_%s" % RandomId(use_static_seed=False).get_unique_id()
```

Comment:
Instead of `"driverid_%s"`, how about `"filterid_%s"`?

Reply:
+1, will be fixed.